concent_api/common/decorators.py

```diff
@@ -85,3 +89,32 @@ def decorated(*args: Any, **kwargs: Any) -> None:
                 return func(*args, **kwargs)
 
         return decorated
+
+
+class retry_on_database_error(object):
+
+    def __init__(self, func: Callable) -> None:
+        self.func = func
+        self.max_number_of_retries = settings.MAXIMUM_NUMBER_OF_RETRIES  # pylint: disable=attribute-defined-outside-init
+
+    def __call__(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
Are you sure this will be called each time in runtime, not only on server initialization?